src/Web/Slack.hs

```diff
@@ -1,7 +1,6 @@
 {-# LANGUAGE DataKinds #-}
 {-# LANGUAGE OverloadedStrings #-}
 {-# LANGUAGE TypeFamilies #-}
```

Comment:
I think `hlint` will tell you when a file has redundant language extensions. I'm not sure if it takes your `default-extensions into account.

```suggestion
{-# LANGUAGE DataKinds #-}
{-# LANGUAGE OverloadedStrings #-}
{-# LANGUAGE TypeFamilies #-}
```